servicetalk-examples/http/debugging/src/main/resources/log4j2.xml

```diff
@@ -28,6 +28,7 @@
     <!-- Additional useful loggers of interest - include relevant loggers in your log4j2.xml file -->
```

Comment:
Is this file needed since the example also has a dependency on test resources? Perhaps also make this change in test resources log4j.xml as well.

Reply:
I believe it's needed here. log4j.xml form test-resources is for tests purposes only, we don't want too much debug level logging there. In this example, it's useful to show what loggers they can enabled for extra visibility.